tests/rpc/test_rpc_apiserver.py

```diff
@@ -1333,7 +1333,7 @@ def test_list_available_pairs(botclient):
     rc = client_get(client, f"{BASE_URI}/available_pairs")
 
     assert_response(rc)
-    assert rc.json()['length'] == 13
+    assert rc.json()['length'] == 15
```

Comment:
doesn't this test mean that the result includes also mark candles?

I don't think that's correct - we pass nothing as candle-type, so i'd assume that none `''` pairs will be removed/filtered out.

Reply:
I added these files to `tests/testdata/`

- XRP_USDT-1h-mark.json
- XRP_USDT-1h.json
- UNITTEST_USDT-1h-mark.json

I think that it added the pairs `XRP_USDT` and `UNITTEST_USDT` got added. I don't really know what this method tests though, I understand the changes I made in this PR only at a shallow level for the most part. We would probably never want the mark price candles without the futures candles, but unless we hard code something that prevents that then we could have mark with the futures candles

Reply:
the way i'd expect the above method to function is that it'll give you one more file (`XRP_USDT-1h.json`) - but NOT the other two (these are mark candles - should not be reported as regular candles).

Reply:
I don't actually understand what the method `list_available_pairs` is for

Reply:
the name is pretty explanatory i think :laughing: 
it lists all available pairs (it's basically what's called when using `freqtrade list-data`).

Reply:
>  it lists all available pairs (it's basically what's called when using freqtrade list-data).

So whats the problem with showing mark pairs then

Reply:
showing mark candles is fine - IF the user elected to see mark candles.

without selection - only spot candles should be shown.